packages/dx-react-grid-bootstrap3/src/templates/table-header-cell.jsx

```diff
@@ -18,13 +21,32 @@ export class TableHeaderCell extends React.PureComponent {
     this.onCellClick = (e) => {
       const { allowSorting, changeSortingDirection } = this.props;
       if (!allowSorting) return;
-      e.stopPropagation();
       const cancelSortingRelatedKey = e.metaKey || e.ctrlKey;
       changeSortingDirection({
         keepOther: e.shiftKey || cancelSortingRelatedKey,
         cancel: cancelSortingRelatedKey,
       });
     };
+    this.handleKeyDown = (e) => {
```

Comment:
This method looks very similar to `onCellClick`. Let's try to refactor the methods mentioned somehow.